pkg/controllers/management/feature/feature_handler.go

```diff
@@ -40,9 +43,25 @@ func (h *handler) sync(_ string, obj *v3.Feature) (*v3.Feature, error) {
 	if obj.Name == features.TokenHashing.Name() {
 		return obj, h.refreshTokens()
 	}
+
+	if obj.Name == features.Harvester.Name() {
+		return obj, h.toggleHarvesterNodeDriver(obj.Name)
+	}
 	return obj, nil
 }
 
+func (h *handler) toggleHarvesterNodeDriver(harvester string) error {
+	lister := h.mgmt.Management.NodeDrivers("").Controller().Lister()
+	cli := h.mgmt.Management.NodeDrivers("")
+	if val := features.GetFeatureByName(harvester).Enabled(); val {
+		m, _ := lister.Get("", harvester)
```

Comment:
We should at least debug log the error here or handle/return it.

Reply:
@nickgerace I just updated the code. Many thanks for the input.